kernel-api/src/test/scala/org/apache/toree/utils/TaskManagerSpec.scala

```diff
@@ -34,7 +34,7 @@ class TaskManagerSpec extends FunSpec with Matchers with MockitoSugar
   with Eventually with Timeouts
 {
   implicit override val patienceConfig = PatienceConfig(
-    timeout = scaled(Span(200, Milliseconds)),
+    timeout = scaled(Span(30, Seconds)),
```

Comment:
Maybe... my conversation with @Lull3rSkat3r was that I feel these tests are trying to be units, but are actually more of an integration test. Either we find a different way to test the functionality without involving Akka, or we moved them as part of overall SystemTest.